invenio_preservation_sync/ext.py

```diff
@@ -30,6 +38,7 @@ def init_app(self, app):
         app.extensions["invenio-preservation-sync"] = self
         if self.is_enabled(app):
             app.register_blueprint(self.preservation_info_resource.as_blueprint())
+            app.register_blueprint(blueprint)
```

Comment:
Quick test: can you try to pass the `template_folder="templates"` in the above `as_blueprint` and see if it works, without declaring and registering yet another blueprint only from the templates?